village/src/components/DirectSmurf.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import axios from "axios";
+import Smurf from "./Smurf";
+
+class DirectSmurf extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      linkedSmurf: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:3333/smurfs")
+      .then(response => {
+        const id = this.props.match.params.id;
+        const smurf = response.data.find(data => {
+          return data.id === id;
```

Comment:
```suggestion
          return data.id === Number(id);
```
The id off the match.params is a string, and the id from the data is a number, so you'll have to compare them like this